src/manifestMap.ts

```diff
@@ -31,7 +31,11 @@ export class ManifestMap {
     }
 
     getFirstItem(): Manifest | undefined {
-        return Array.from(this.inner.values())[0]
+        for (const [, manifest] of this.inner) {
+            return manifest
+        }
+
+        return undefined
```

Comment:
I am not sure about this change, what are the benefits of  going through a for if we can get the first item from the map?

Reply:
So constructing an array won't be needed?

Reply:
You can do that with `const key = this.inner.keys().next().value; return this.inner[key]` no?

Reply:
I did that before, but we need to cast it to string for eslint to not complain

Reply:
```ts
const key = this.inner.keys().next().value as string | undefined

if (key !== undefined) {
     return this.inner.get(key)
}

return undefined
```

The for loop seems more elegant for me

Reply:
![image](https://user-images.githubusercontent.com/64297935/158160293-e2f8a395-536f-40a0-bce8-ea2ff62116d7.png)

it's complaining here :shrug:

Reply:
I would drop this commit for now then, i will give it a try